ssn/frag/Introduction.html

```diff
@@ -0,0 +1,61 @@
+<!--  <section id="Introduction" class="informative">  -->
+<h2>Introduction</h2>
+
+<p>
+  Sensors are a major source of data available on the Web today.
+  However, searching, reusing, integrating, and interpreting sensor data requires more than just the sensing results.
+  For the proper interpretation of these values, knowledge of the feature of interest, such as a river, the observed
+  property, such as flow velocity, the sampling strategy, such as the specific locations and
+  times at which the velocity was measured, and a variety of other information are required.
+</p>
+<p>
+  OGC's Sensor Web Enablement standards [[SWE]], [[SensorML]], [[OandM]], [[OMS]], [[STA]] provide
+  conceptual models, as well as some encodings for the annotation of sensors and their observations.
```

Comment:
suggestion to add : "Section 7 of [[OMS]] provides a comprehensive description of the characteristics of observations and samples"
see : https://docs.ogc.org/as/20-082r4/20-082r4.html#_fundamental_characteristics_of_observations_and_samples_informative

Reply:
I omitted 'Section 7 of' - too much for the introduction. 
But I added the specific pointer to Section 7 in the Origins narrative.